maven/src/main/java/org/commonjava/maven/galley/maven/ArtifactManager.java

```diff
@@ -80,10 +80,10 @@ Transfer store( Location location, ArtifactRef ref, InputStream stream )
     Transfer store( Location location, ArtifactRef ref, InputStream stream, EventMetadata eventMetadata )
         throws TransferException;
 
-    boolean publish( Location location, ArtifactRef ref, InputStream stream, long length )
+    boolean publish( Location location, ArtifactRef ref, InputStream stream, long length, EventMetadata eventMetadata )
```

Comment:
This one is a little bit weird. It's a subsection of galley that is maven-specific, so I think it would be okay to leave the EventMetadata parameter out (if we were just using it for package type)...but then again, we use EventMetadata for other things now, so it's reasonable to have it here. I guess let's leave it like you have it.